### ISSUE TITLE
Documentation clarification for EcoRoof module

### ISSUE THREAD

**Issue by BobHenninger** [NREL/EnergyPlus#4416]
Helpdesk Ticket 9209

This could be added to the docs, assuming I'm correct. Hard to follow what's happening.
 Michael J. Witte on 14 March 2014 06:51 PM

Posted on: 14 March 2014 06:51 PM
It is the temperature at the interface between the soil and the next material layer. And this becomes the soil temperature for the next timestep. You can see this by reporting Surface Outside Face Temperature and Green Roof Soil Temperature at the Timestep reporting frequency.
 Mike

> Dear helpdesk responsible,
> I'm currently carrying out a series of simulations about the use of
> green 
> roofs by using the EcoRoof model implemented in EnergyPlus v.7.0.
> For the sake of clearness, I would like to know if the outer surface 
> temperature that I require as a output for the roof surafces is that
> of the 
> greenery (i.e. the outermost layer of the green roof) or that of the
> outermost 
> layer of the construction immediately below the green roof (this is
> what I 
> suppose).
> Actually, the EnergyPlus guide on the EcoRoof model does not highlight
> this 
> aspect.
> Thank you for your cooperation.
> Best regards
> 
> Vincenzo Costanzo

**Comment by Myoldmopar** [comment]
Closed with #7331
